test/acceptance/features/bindAppToServiceUsingSecret.feature

```diff
@@ -761,3 +761,202 @@ Feature: Bind values from a secret referred in backing service resource
+          """
+          apiVersion: binding.operators.coreos.com/v1alpha1
+          kind: ServiceBinding
+          metadata:
+              name: bind-provisioned-service-4
+          spec:
+              services:
+              - group: ""
+                version: v1
+                kind: Secret
+                name: provisioned-secret-1
+              application:
+                name: myaop-provision-srv
+                group: apps
+                version: v1
+                resource: deployments
+          """
+        * jq ".status.conditions[] | select(.type=="Ready").status" of Service Binding "bind-provisioned-service-4" should be changed to "False"
+        * The Secret is present
+            """
+            apiVersion: v1
+            kind: Secret
+            metadata:
+                name: provisioned-secret-1
+            stringData:
+                username: foo
+                password: bar
+            """
+
+        Then Service Binding "bind-provisioned-service-4" is ready
+        And Content of file "/bindings/bind-provisioned-service-4/username" in application pod is
+            """
+            foo
+            """
+        And Content of file "/bindings/bind-provisioned-service-4/password" in application pod is
+            """
+            bar
+            """
+
+    Scenario: Inject binding to an application from two Secret resources referred as services
+        Given Generic test application "myaop-provision-srv" is running
+        When Service Binding is applied
+          """
+          apiVersion: binding.operators.coreos.com/v1alpha1
+          kind: ServiceBinding
+          metadata:
+              name: bind-provisioned-service-1
+          spec:
+              services:
+              - group: ""
+                version: v1
+                kind: Secret
+                name: provisioned-secret-1
+                id: one
+              - group: ""
+                version: v1
+                kind: Secret
+                name: provisioned-secret-2
+                id: two
+              application:
+                name: myaop-provision-srv
+                group: apps
+                version: v1
+                resource: deployments
+          """
+        * The Secret is present
+            """
+            apiVersion: v1
+            kind: Secret
+            metadata:
+                name: provisioned-secret-1
+            stringData:
+                username: foo
+                password: bar
+            """
+        * The Secret is present
+            """
+            apiVersion: v1
+            kind: Secret
+            metadata:
+                name: provisioned-secret-2
+            stringData:
+                username2: foo2
+                password2: bar2
+            """
+
+        Then Service Binding "bind-provisioned-service-1" is ready
+        And jq ".status.secret" of Service Binding "bind-provisioned-service-1" should starts with "bind-provisioned-service-1"
```

Comment:
we should not advertise this kind of info to users - it is not a part of api/contract - the spec only says that this field will be populated with the intermediate secret name, but does not say anything about naming convention.

Reply:
See the last sentence in this section: https://github.com/k8s-service-bindings/spec#service-binding
It reads like this:
>  A Service Binding resource SHOULD reflect the secret projected into the application as `.status.binding.name`.

That's a recommendation.

Reply:
it says only that the secret name should be set in `.status.binding.name`, but it does not force any particular naming convention.

Reply:
Done. 832c72fcec87d9f227ff5ea071ff759a112edc95